PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/util/PluginUtil.java

```diff
@@ -178,14 +178,18 @@ public static Icon getIcon(String iconPath) {
     }
 
     public static Icon getIcon(String iconPath, int width, int height) {
-        Image azureImage = IconLoader.toImage(PluginUtil.getIcon(iconPath));
-        Image result = azureImage.getScaledInstance(width, height, java.awt.Image.SCALE_SMOOTH);
-        return new ImageIcon(result);
+        try {
+            Image azureImage = IconLoader.toImage(PluginUtil.getIcon(iconPath));
+            Image result = azureImage.getScaledInstance(width, height, java.awt.Image.SCALE_SMOOTH);
+            return new ImageIcon(result);
+        } catch (ArrayIndexOutOfBoundsException e) {
+            return null;
```

Comment:
Log the stacktrace here might be better

Reply:
Move the catch to survey dialog and send telemetries.